arangod/IResearch/IResearchLink.cpp

```diff
@@ -1983,6 +2041,86 @@ IResearchViewStoredValues const& IResearchLink::storedValues() const noexcept {
   return _meta._storedValues;
 }
 
+void IResearchLink::removeStats() {
+  if (_linkStats) {
```

Comment:
maybe get a reference to `MetricsFeature` only once?

Reply:
Done